src/pages/smart-contracts/clarity-values-guide.md

```diff
@@ -0,0 +1,143 @@
+---
```

Comment:
this page isn't in the navigation yet. can you make a new entry here: https://github.com/blockstack/docs/blob/master/src/common/navigation.yaml#L8

Reply:
The filename of this file needs to match the entry in the yaml file. Should update it to `clarity-values`